docs/quick-start/framework/java-spring-boot/README.mdx

```diff
@@ -0,0 +1,329 @@
+---
+slug: /quick-start/java-spring-boot
+sidebar_label: Java Spring Boot
+sidebar_custom_props:
+  description: Spring Boot is a web framework for Java that enables developers to build secure, fast, and scalable server applications with the Java programming language.
+  logoFilename: 'spring-boot.svg'
```

Comment:
Use `spring-web.svg`?

Reply:
The logo is actually for spring-boot, in case of any future reference should not named as spring-web.